locals.tf

```diff
@@ -159,7 +159,10 @@ locals {
   )
   enable_container_app_blob_storage                = var.enable_container_app_blob_storage
   container_app_blob_storage_public_access_enabled = var.container_app_blob_storage_public_access_enabled
-  container_app_blob_storage_ipv4_allow_list       = var.container_app_blob_storage_ipv4_allow_list
+  container_app_blob_storage_ipv4_allow_list = concat(
+    [jsondecode(azapi_resource.default.output).properties.outboundIpAddresses[0]],
```

Comment:
This can be changed to just:
```
jsondecode(azapi_resource.default.output).properties.outboundIpAddresses
```
(Not as a list)
At least then if there ever happens to be more than 1 outbound ip address, it'll be picked up